src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -21,17 +22,30 @@
  */
 class MesosCloud extends AbstractCloudImpl {
 
+  private static final Logger LOGGER = Logger.getLogger(MesosCloud.class.getName());
```

Comment:
See my comment below. We should use slf4j:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(MesosCloud.class);
```